internal/legacy/gateway/pf_structs.go

```diff
@@ -379,13 +380,14 @@ func (d DatR) MarshalJSON() ([]byte, error) {
 func (d *DatR) UnmarshalJSON(data []byte) error {
 	i, err := strconv.ParseUint(string(data), 10, 32)
 	if err != nil {
-		d.LoRa = strings.Trim(string(data), `"`)
+		d.LoRa = strings.Trim(string(data), `"`) //"
```

Comment:
the comment looks strange, is well understood what the `Trim` function does.

Reply:
Just closing open " because it's confusing highlighter, etc...